src/vs/workbench/contrib/terminalContrib/accessibility/browser/terminalAccessibleWidget.ts

```diff
@@ -118,11 +118,6 @@ export abstract class TerminalAccessibleWidget extends DisposableStore {
 					// On escape, hide the accessible buffer and force focus onto the terminal
 					this.hide(true);
 					break;
```

Comment:
I don't think we need to hide it on `Tab`

Reply:
If focus moves out of the accessible buffer we do, depends on what tab does

Reply:
in a read only editor, tab always moves focus to the next tabbable element. but now seems like we shouldn't handling this per key and instead on blur of the widget.

Reply:
that's not how it was working before, but I think this is a good step in the direction of the accessible view, since that hides on blur